modules/configure-alert-data-retention.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-alert-data-retention.adoc
+:_module-type: PROCEDURE
+[id="configure-alert-data-retention_{context}"]
+= Configuring alert data retention
+
+You can configure alert retention settings by using the RHACS portal.
+
+.Prerequisites
+
+* You need the `Config` role with `read` and `write` permissions to configure data retention.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *System Configuration*.
+. On the *System Configuration* view header, click *Edit*.
+. Under the *Data Retention Configuration* section, update the number of days for each type of data:
+* All Runtime Violations
```

Comment:
Should these be in bold like they are in the original stackrox docs, because they are UI elements? Otherwise, the all caps looks a little weird (especially with "For" capitalized)